docs/source/cdk/aws-setup.rst

```diff
@@ -0,0 +1,45 @@
+AWS Setup
+=========
+Download/requirements
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
AWS Setup
=========

Download/requirements
~~~~~~~~~~~~~~~~~~~~~
```
Missed the newline here, but added it below. Here is an example of how Numpy's layout is with plenty of newlines between different entities and levels.
https://github.com/numpy/numpy/blob/48a8277855849be094a5979c48d9f5f1778ee4de/doc/source/index.rst?plain=1#L1-L7